spynnaker/pyNN/models/neuron/synapse_dynamics/abstract_synapse_dynamics.py

```diff
@@ -398,3 +409,11 @@ def validate_connection(
         # By default, just ask the connector
         synapse_info.connector.validate_connection(
             application_edge, synapse_info)
+
+    @property
+    @abstractmethod
+    def synapses_per_second(self) -> int:
+        """
+        Approximate number of synapses that can be processed per second
```

Comment:
Would be good to add a comment about which it does under estimate, over estimate or as close as possible.

Reply:
Should that not be the other way around.  Underestimating will create a system that can not handle the received spikes

Reply:
No, if you underestimate how many spikes you can process, you can process more than you estimate - that is, your estimate is under what you can really handle.  If you overestimate you are guessing you can do more than you really can, so more spikes will be sent to you than you can handle.